library/src/main/kotlin/com/oddin/oddsfeedsdk/api/entities/sportevent/SportEvent.kt

```diff
@@ -51,7 +60,19 @@ data class Scoreboard(
     val currentDefenderTeam: Int?,
     val homePoints: Int?,
     val awayPoints: Int?,
-    val remainingGameTime: Int?
+    val remainingGameTime: Int?,
+    val homeRuns: Int?,
+    val awayRuns: Int?,
+    val homeWicketsFallen: Int?,
+    val awayWicketsFallen: Int?,
+    val homeOversPlayed: Int?,
+    val homeBallsPlayed: Int?,
+    val awayOversPlayed: Int?,
+    val awayBallsPlayed: Int?,
+    val homeWonCoinToss: Boolean?,
+    val homeBatting: Boolean?,
+    val awayBatting: Boolean?,
+    val inning: Int?
```

Comment:
**[BETTER-FOR-FUTURE-REVIEWS]:** I believe, that C# allows trailing commas everywhere. Then it would be better to keep using them. 

> The Kotlin style guide encourages the use of trailing commas at the declaration site and leaves it at your discretion for the call site. [Kotlin coding conventions](https://kotlinlang.org/docs/coding-conventions.html#trailing-commas)

One might try:

```suggestion
    val inning: Int?,
```

Multiple occurrences.